matlab_simulation/05-estimation/kalman_filter/measurement_model.m

```diff
@@ -0,0 +1,53 @@
+function [C,D,Q,m] = measurement_model(example)
```

Comment:
@rhyse-me780 Yes I agree with your points however, the reasoning behind coding it this way is what @nanwei1 mentioned. 
Besides this, I thought it would easy comparison of the models. That way if they have to develop a similar model, they have the motion and measurement models grouped together, rather than clicking on the respective examples.